auraed/src/lib.rs

```diff
@@ -51,36 +51,30 @@
     while_true
 )]
 #![warn(missing_debug_implementations,
-        // TODO: missing_docs,
-        trivial_casts,
-        trivial_numeric_casts,
-        unused_extern_crates,
-        unused_import_braces,
-        unused_results
-        )]
+// TODO: missing_docs,
```

Comment:
why did the indent change here?  not a big deal just curious if my autoformat is doing it wrong.

Reply:
This has to be my or Nóva's automatic formatting (I assume ours are the same since we're both on clion). I edited it to put it back in a cleanup.